website/content/labs/lab5.md

```diff
@@ -183,9 +183,10 @@ As above, you should use the TDD workflow when implementing these new functions.
 
```

Comment:
Again, can't comment on the right line, but line 176 is missing a semicolon at the end

`circle_t *c = circle_new(point_new(0, 0), 5);`

Reply:
Fixed